js/interface.js

```diff
@@ -3,6 +3,8 @@ Fliplet().then(function() {
   var appId = Fliplet.Env.get('appId');
   var dataSourceProvider = null;
   var $dataColumnsEmail = $('#emailColumn');
+  var $loggedInUserTime = $('[name="loggedInUserTime"]');
+  var timeInMinutes = 60;
```

Comment:
The time of what in minutes? `minutesInHour`?

Reply:
Actually its `minutesInHour`. I did the changes accordingly. It's my bad